lib/toolbox/set_name.c

```diff
@@ -0,0 +1,19 @@
+#include "set_name.h"
+#include <stdio.h>
+#include <stdint.h>
+#include <furi_hal_rtc.h>
+
+void set_name(char* name, uint8_t max_name_size, char* app_name) {
```

Comment:
I'd rather see one universal interface for random and detailed api with rtc flag check on the inside